data/json/recipes/recipe_food.json

```diff
@@ -1002,6 +1002,25 @@
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
     "components": [ [ [ "dahlia_root", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "salsify_baked",
```

Comment:
This seems like an easy recipe, perhaps add `"autolearn": true`, and bump down the level the books teach you it?

Reply:
Come to think of it, why do baked Dahlia Roots require a cookbook to understand?